lib/js-yaml/loader.js

```diff
@@ -852,7 +850,7 @@ function readBlockScalar(state, nodeIndent) {
 
       // Just one line break - perceive as the same line.
       } else if (emptyLines === 0) {
-        if (didReadContent) { // i.e. only if we have already read some scalar content.
+        if (contentNewline) { // i.e. only if we have already read some scalar content.
```

Comment:
Or `!== ''`. I'd expect all of them to have about the same performance, since the type of `contentNewline` is fully determined as non-null string at compile time.